composer.json

```diff
@@ -272,6 +275,10 @@
             "@php -d date.timezone=UTC -d memory_limit=-1 ./vendor/phpunit/phpunit/phpunit -c tests/Integration/phpunit.xml",
             "@php -d date.timezone=UTC ./vendor/phpunit/phpunit/phpunit -c tests/Integration/phpunit-isolated.xml"
         ],
+        "phpstan": [
+            "Composer\\Config::disableProcessTimeout",
+            "make phpstan"
```

Comment:
I think that composer scripts shouldn't depend on `make`. The previous version was ok for me (`./vendor/bin/phpstan analyse --memory-limit 1G -c phpstan.neon.dist`). 

`make` is not available on each platform. If I use composer scripts, I don't have to know if I have `make` or something else. 

I don't know if I'm clear 😉

Reply:
Yeah, its a bit unclear now that there are two different places for these. 

I will raise this on the slack channel and try to get somekind of "vote" if Makefile should be deprecated and all scripts moved into composer scripts.